install.md

```diff
@@ -270,15 +270,12 @@ Then to install Buildah on Ubuntu follow the steps in this example:
 
 ### Debian
 
-To install the required dependencies, you can use those commands, tested under Debian GNU/Linux amd64 9.3 (stretch):
+To install the required dependencies on Debian you can use these commands, tested under Debian GNU/Linux amd64 12 (bookworm):
 
 ```
-gpg --recv-keys 0x018BA5AD9DF57A4448F0E6CF8BECF1637AD8C79D
-sudo gpg --export 0x018BA5AD9DF57A4448F0E6CF8BECF1637AD8C79D >> /usr/share/keyrings/projectatomic-ppa.gpg
-sudo echo 'deb [signed-by=/usr/share/keyrings/projectatomic-ppa.gpg] http://ppa.launchpad.net/projectatomic/ppa/ubuntu zesty main' > /etc/apt/sources.list.d/projectatomic-ppa.list
-sudo apt update
-sudo apt -y install -t stretch-backports golang
-sudo apt -y install bats btrfs-tools git libapparmor-dev libglib2.0-dev libgpgme11-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man
+  sudo apt-get -y -qq update
+  sudo apt-get -y install bats btrfs-progs git libapparmor-dev libglib2.0-dev libgpgme11-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man make
+  sudo apt-get -y install golang
```

Comment:
Does the golang install need to be separate? I guess you can combine the golang install with the other packages in the previous line. Apart from that, LGTM

Reply:
I suppose not, I just kept it like that because it was that way before. I'll update.

Reply:
yes, skopeo has changed name. I've already updated that locally - it will be included in the updated commit.